PPL/costAnalisis.md

```diff
@@ -0,0 +1,1724 @@
+   <td colspan="2" ><strong>Costes posteriores de la ONG (por año)</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Coste del dominio
+   </td>
+   <td><strong>50€</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Coste del despliegue
+   </td>
+   <td><strong>876€</strong>
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Total</strong>
+   </td>
+   <td><strong>926€</strong>
+   </td>
+  </tr>
+</table>
+
+
+<br/>
+
+# 4. TCO
+
+Para el correcto análisis del TCO tendremos en cuenta los costes asociados al desarrollo y los costes de la puesta en producción de la aplicación durante  los dos primeros años de vida (15 semanas de desarrollo + **89 semanas**).
+
+Para ello calculamos los costes salariales del equipo encargado del mantenimiento y actualización de la aplicación una vez puesta en producción, se contará con un equipo de 5 desarrolladores junior y un senior con una jornada laboral de 10 horas semanales.
+
+También tendremos en cuenta las compensaciones por trabajo en remoto de los trabajadores y los costes de las licencias requeridas.
+
+Por último, se ha analizado y tenido en cuenta para el TCO, los costes de marketing y publicidad, utilizando el mismo personal externo contratado durante el periodo de desarrollo, con las mismas condiciones laborales, es decir, con un contrato de 10h semanales, y con la misma retribución.
+
+
+<table>
+  <tr>
+   <td colspan="2" ><strong>Costes totales de propiedad</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Costes totales del desarrollo
+   </td>
+   <td><strong>54.920,48€</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Reservas de contingencia
+   </td>
+   <td><strong>5.492,05€</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Costes totales en producción (98 semanas)
+   </td>
+   <td><strong>96.618,61€ (44.175,38€/37 semanas)</strong>
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Total</strong>
+   </td>
+   <td><strong>157.031,14€</strong>
+   </td>
+  </tr>
+  <tr>
+   <td><strong>TCO (Mes)</strong>
+   </td>
+   <td><strong>3271,50€</strong>
+   </td>
+  </tr>
+</table>
+
+
+Si además queremos ver el Capex y el Opex, vamos a calcular lo que equivale a cada uno. Lo primero que vamos a ver es:
+
+- **Capex:** Representa el gasto inicial necesario para adquirir, construir o mejorar los activos físicos de un proyecto. Incluye la compra de equipos, maquinaria, infraestructura, y otros activos que se capitalizan y se amortizan a lo largo del tiempo. Los gastos de CAPEX son típicamente de naturaleza no recurrente y se consideran como inversiones a largo plazo en el negocio.
+- **Opex:** Representa los costos recurrentes y continuos asociados con el funcionamiento y mantenimiento de los activos adquiridos con el CAPEX. Incluye gastos como salarios del personal, mantenimiento, suministros, alquileres, servicios públicos, y otros costos operativos necesarios para mantener en funcionamiento el proyecto. Los gastos de OPEX son considerados como gastos corrientes y se incurren durante toda la vida útil del proyecto.
+
+Para el Capex:
```

Comment:
Finally, update the "Sueldos menusales en desarrollo" in Capex and add a line for GDPR in Opex

Reply:
is updated already because Community manager has the same salary as junior!